### PR TITLE
My first robot improvements

### PR DESCRIPTION
**Pull request by mikaukora**
Update rpaframework to access the latest keywords.
Use Screenshot to allow both image in log.html and a separate file as artifact.
Add keyword to open cloud in the final step to verify cloud account.
Clarify instructions about cloud linking.

### CODE REVIEW

**Review by vermario — APPROVED** [review]
I pushed some commits: 
- added `robot.yaml` (tested with Lab nightly build)
- renamed to `tasks.robot` 
- added link to the page explaining how to do the linking to cloud
- small edits (commas, etc)

File: my-first-robot/robot.yaml [hunk 1]
```diff
@@ -0,0 +1,20 @@
+tasks:
+  Image search:
+    command:
+      - python
+      - -m
+      - robot
+      - -d
```
Comment (mikaukora) [inline my-first-robot/robot.yaml:7]: Let's use 
`      - --report
      - NONE`
here to prevent report.html being generated.

Comment (vermario) [reply my-first-robot/robot.yaml:7]: I tried adding that line (both with `- NONE` and just with `NONE`, and with that Robocorp Lab (nigthly, that uses `robot.yaml`) sees the run as failed: 

report none

Comment (vermario) [reply my-first-robot/robot.yaml:7]: does it need to appear at a specific point of the command? or is the syntax wrong?

Comment (vermario) [reply my-first-robot/robot.yaml:7]: and yes! of course each element on its own line. Fixing!

**Comment by mikaukora** [comment]
Entrypoint scripts still use old robot.robot name. That needs to be updated.

**Comment by mikaukora** [comment]
Good to go!

**Pull request merged**